posthog/tasks/utils.py

```diff
@@ -29,8 +29,7 @@ class CeleryQueue(Enum):
     DEFAULT = "celery"
     STATS = "stats"
     EMAIL = "email"
-    INSIGHT_EXPORT = "insight_export"
-    INSIGHT_REFRESH = "insight_refresh"
+    LONG_RUNNING = "longrunning"  # any task that has a good chance of taking more than a few seconds should go here
```

Comment:
```suggestion
    LONG_RUNNING = "long_running"  # any task that has a good chance of taking more than a few seconds should go here
```

Right? 🙈 

Maybe we should make all worker names != any queue names to reduce confusion?

Reply:
Lifesaver!

Yes, I think that makes sense in the future, although it sounds like an annoying change to make now.

Add new queues names in chart, deploy, change them here, deploy, remove old ones in chart? Will consider going forward.

Reply:
>it sounds like an annoying change to make now.

Agreed... that thought came slightly too late.